tune/cli.py

```diff
@@ -371,15 +371,15 @@ def local(  # noqa: C901
                     )
                 root_logger.info(f"Current optimum:\n{best_point_dict}")
                 root_logger.info(
-                    f"Estimated value: {np.around(best_value, 4)} +- "
-                    f"{np.around(best_std, 4).item()}"
+                    f"Estimated Elo: {np.around(-best_value * 100, 4)} +- "
+                    f"{np.around(best_std * 100, 4).item()}"
                 )
                 confidence_val = settings.get("confidence", confidence)
                 confidence_mult = erfinv(confidence_val) * np.sqrt(2)
                 root_logger.info(
                     f"{confidence_val * 100}% confidence interval of the value: "
```

Comment:
Also change `value` to `Elo score` or `Elo value`

Reply:
I *think* I've changed what you asked. I used `Elo value` to be consistent with the internal naming.